module_7/assignment2_3.js

```diff
@@ -0,0 +1,115 @@
+ function sort(array, key, order) {
+   const sortOrder = order === "asc" ? 1 : order === "desc" ? -1 : null;
+   return array.sort((a, b) => {
+     const valueA = a[key];
+     const valueB = b[key];
+
+     if (valueA < valueB) {
+       return -sortOrder;
```

Comment:
No need to do add if else statement is sorting is done by number data type
users.sort((a, b) => b.id - a.id); // descending ad for assign a then replace b
and we need sorting by string 
(a, b) => a.first_name < b.first_name   ? -1 : 1